src/shared/components/DeleteDataForm/TimeRangeDropdown.tsx

```diff
@@ -50,7 +51,10 @@ const TimeRangeDropdown: FC<Props> = ({
 
   return (
     <div ref={buttonRef} className={className} data-testid="timerange-dropdown">
-      <Dropdown.Button onClick={() => setPickerActive(!pickerActive)}>
+      <Dropdown.Button
+        icon={IconFont.Clock_New}
+        onClick={() => setPickerActive(!pickerActive)}
```

Comment:
if we're already here:

```suggestion
        onClick={() => setPickerActive(prevPicker => !prevPicker)}
```